hw2/04-background.js

```diff
@@ -1 +1,32 @@
 // Add your code here
+const body = document.getElementById('background');
+const timeInterval = document.getElementById('interval');
+const changeColorBtn = document.getElementById('changeColorBtn');
+
+const colorChange = function randomColorChangeForBodyBackground() {
+  const r = Math.floor(Math.random() * 256);
+  const g = Math.floor(Math.random() * 256);
+  const b = Math.floor(Math.random() * 256);
+  const a = Math.floor(Math.random() * 10) / 10;
```

Comment:
I believe generating random number for alpha might not work out in this case because it can be between 0.0 to 0.9. Our purpose is to make random color dim, setting alpha as 0.5 might help to do that.

Reply:
I was interested in seeing how it would act with a random alpha and forgot to set it to a lower value. Thanks!